### ISSUE TITLE
Built-in thermals

### ISSUE THREAD

**Issue by AgentRev** [A3Armory/A3A_A3Wasteland_Sock.Altis#58]
I would like you guys to try disabling built-in thermals for a couple days and gauge the reaction. I've seen a lot of players whose sole strategy is to whip out their designator and scan the horizon as soon as they're getting shot, all while dancing on top of a hill.

_Are you hidden in a bush, deep in the valley, wearing a full ghillie? Nooooope, you're actually dead, because that random guy 1km away had a designator._

This is getting pretty ridiculous.

**Comment by mooneymoon** [comment]
The consensus has been to leave the laser designator. It's been discussed a few times already.

**Comment by tylers-dev** [comment]
The LD is the last remaining thermal in the server. It's LoS limitation is a pretty fair playing ground. If they can see you, you can see them. Our server has gone back and forth a lot on thermals. It feels like one of those determining factors for separating a regular server from a hardcore server. A lot of people want to make the gameplay harder but realize harder isn't popular to the masses. 

I also don't see how it can hurt to try disabling them for a week or so. But the reaction feedback would likely come from our forums, who, like Mooney said, most still support keeping the LD.

**Comment by mooneymoon** [comment]
I am dead-set against making this change without a poll.

**Comment by tylers-dev** [comment]
A script limiting idea, it would be cool if the LD thermal had to use LD batteries. Thermal vision on the LD would be dependent on there being an LD battery. Then by storing the amount of time spent in thermal vision we could reach a threshold where the battery is deleted (depleted). Say 5mins of use. Which is a lot when you do just quick scans. No idea if possible, just an interesting thought.

**Comment by mooneymoon** [comment]
I wouldn't be opposed to that. In addition, sell a version in the store without batteries.

**Comment by codexGW** [comment]
There's a couple contentious threads about this already. I think something like this should be brought to a forum vote, to not alienate the donator base. There's also the issue with titans not having NV. I think the best way to address the LD is to make the movement a bit clunky and slow when and right after using it like the titans. If that's possible.

**Comment by AgentRev** [comment]
> The LD is the last remaining thermal in the server.

That's not true, regular thermal scopes spawn in some mission crates, I have 2 TWS in private storage. Titans have built-in thermal too. Remote designators and UAV planes have thermals which are not affected by this option.

**Comment by AgentRev** [comment]
> A script limiting idea, it would be cool if the LD thermal had to use LD batteries. Thermal vision on the LD would be dependent on there being an LD battery. Then by storing the amount of time spent in thermal vision we could reach a threshold where the battery is deleted (depleted).

That would be great, except that scripts do not possess the capability to detect the presence of batteries loaded in the LD. Completely ignored by BIS like always: https://feedback.bistudio.com/T79976

That's why I auto-add batteries on LD purchase and player save restore.

Also, you can't "link" a depletion timer to the LD batteries, it would have to be linked to the player unit, and so such a timer wouldn't update if teammates swap LDs or the player buys or finds a LD. The player could also have multiple LDs in his backpack, that would be a real mess.

**Comment by AgentRev** [comment]
Alternatives other than disabling thermal include:
- Removing LDs from genstores; there is always at least one in `mission_USLaunchers` crates, and can be increased if necessary.
- Increasing LD price; I recently dropped the vanilla price to 250, as built-in thermal is disabled by default. However, if you keep it enabled, it would make sense to increase it to like 2000, so any fresh spawn can't just go grab one outright. This however doesn't really solve the problem, as people who know how to use it best usually have a shitload of money.

**Comment by codexGW** [comment]
There's thermal on the titans to. People use those for thermal a lot and you cant remove thermal from those because they are worthless at night (no NV). Creampie added some wonky thing to prevent thermal removal for titans at night to his original thermal removal script.

If you want to put the LD to mission only like the other thermals we should maybe put that to a vote. It seems like all the people who have the thermal scopes now are regulars and they don't seem in short supply.

**Comment by codexGW** [comment]
Here ya go.
http://www.a3armory.com/forum/thread-611.html

**Comment by AgentRev** [comment]
If they stay in store I do hope you'll increase the price at least

**Comment by AgentRev** [comment]
Well, from the vote and comments it definitely is a controversial topic. I believe the whole "top players are gonna hoard them" argument is exaggerated, but whatever.

**Comment by codexGW** [comment]
I'll increase the price to 2500 next pbo update. When the voting is done I will only count the people with post as well.

**Comment by codexGW** [comment]
The vote passed barely. http://www.a3armory.com/forum/thread-611-post-4551.html#pid4551

**Comment by tylers-dev** [comment]
So something isn't adding up with that poll. I know for a fact I saw the count at one point was 20-20, then it was 21-20. But when I count the names its 21-18. ?
Oh and another thing I'm curious about. Is it fair to count the votes for the accounts created on the day of the poll or have 0 posts made? I've found 5 people in this poll who have never contributed anything to the forums other than this vote and/or created their account just last week. We've seen this happen with other votes to sway a decision. Anybody can create different accounts from different IPs, and basically rig a poll. We need better prevention for this. That's why I think people who obviously only logged on or created an account for this vote should need a minimum contribution. At least ONE other post you have made, something to prove you are more than an email account and a click.

**Comment by codexGW** [comment]
I removed the 2 new accounts with the same IP, both with 0 post. It was 1 vote on the LD and 2 one the LMG. Or maybe it was the LD with 2. Mooney helped me with it so you can ask him if you think something is up. Go ahead and count the voters post and threads if you want.

**Comment by tylers-dev** [comment]
Cavscout, Mattman, One Angry Grunt, Walipino, Zippy
These 5 have never made a post. To be fair I recognize a few of these names as real players on the server. I'm not asking for a recount, I honestly don't care which way this vote goes. But I'd like to know in the future our polls are as accurate as possible. And I think a nice rule of seeing at least 1 shread of human interaction in the forums means you aren't just an empty account that never plays on the server.
